src/frontend/src/pages/HomePage/components/InfoBlock.tsx

```diff
@@ -1,7 +1,12 @@
-import { Grid, Typography, ListItem, List, useTheme } from '@mui/material';
+import { Grid, Typography, ListItem, List, useTheme, Button } from '@mui/material';
 import { Box } from '@mui/system';
+import { Link } from 'shared';
 
-const InfoBlock: React.FC = () => {
+interface InfoBlockProps {
+  links: Link[];
+}
+
+const InfoBlock: React.FC<InfoBlockProps> = ({ links }) => {
```

Comment:
Can you change the name of this section to OnboardingInfoSection instead, sorry about that I realized this naming is a little confusing

Reply:
You should chnage the name of the file as well to that^